cmd/tfimport/main.go

```diff
@@ -16,7 +16,7 @@
 // Requires Terraform to be installed and for authentication to be configured for each provider in the Terraform configs provider blocks.
 //
 // Usage:
-// $ go run . [--input_dir=/path/to/config]
+// $ go run . [--input_dir=/path/to/config] [--resources 'google_storage_bucket.bucket' --resources 'google_resource_manager_lien.lien']
```

Comment:
I was actually thinking of limiting just based on the resource type e.g. google_storage_bucket. How does one use this when the resource is in a module? Do they need to give a whole module path?

I think if people have to figure out the resource name/full path then it makes the use of tfimport not as useful, since at that point you could just run the `terraform import` command yourself. I would prefer the resource type limit, that should solve the main use cases I imagine...

Reply:
You're right, I've changed it to resource type.